tests/pipeline/test_blaze.py

```diff
@@ -1377,7 +1393,87 @@ def test_deltas(self, asset_info, add_extra_sid):
                 compute_fn=np.nanmax,
             )
 
-    @with_extra_sid
+    @with_ignore_sid()
+    def test_deltas_before_index_0(self, asset_info, add_extra_sid):
+        df = pd.DataFrame(np.array(
+            [],
+            dtype=[
+                ('sid', 'int64'),
+                ('value', 'float64'),
+                ('asof_date', 'datetime64[ns]'),
+                ('timestamp', 'datetime64[ns]'),
+            ],
+        ))
+        expr = bz.data(df, name='expr', dshape=self.dshape)
+
+        T = pd.Timestamp
+        deltas_df_single_sid = pd.DataFrame({
+            'value': [0.0, 1.0, 2.0, 3.0],
+            'asof_date': [
+                T('2013-12-01'),
+                T('2013-12-15'),
+                T('2013-12-02'),  # not more recent than the previous day
+                T('2013-12-16'),
+            ],
+            'timestamp': [
+                T('2014-01-01 23:00'),
+                T('2014-01-02 23:00'),
+                T('2014-01-03 23:00'),
+                T('2014-01-04 23:00'),
+            ],
+        })
+        deltas_df = pd.concat([
+            deltas_df_single_sid.assign(
+                sid=sid,
+                value=deltas_df_single_sid.value + (100 * n),
+            )
+            for n, sid in enumerate(asset_info.index)
+        ])
+        deltas = bz.data(deltas_df, name='deltas', dshape=self.dshape)
+
+        expected_views_single_sid = keymap(pd.Timestamp, {
+            '2014-01-02': np.array([[0.0],
+                                    [0.0]]),
+            '2014-01-03': np.array([[1.0],
+                                    [1.0]]),
+            '2014-01-04': np.array([[1.0],
+                                    [1.0]]),
+            '2014-01-05': np.array([[3.0],
+                                    [3.0]]),
+        })
+
+        column_constant = np.arange(len(asset_info)) * 100
+        expected_views = {
+            k: v + column_constant
+            for k, v in expected_views_single_sid.items()
+        }
+        with tmp_asset_finder(equities=asset_info) as finder:
+            nassets = len(asset_info)
+            expected_max = np.array([0, 1, 1, 3]) + (nassets - 1) * 100
```

Comment:
The `max` calculations here aren't really relevant to the behavior under test here; can we just pass a sentinel or something for the case where we don't care about the pipeline calculation itself?